src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1089,15 +1089,16 @@ class DocBaseViewer extends BaseViewer {
 
             this.controls.render(
                 <DocControls
+                    annotationColor={this.annotationModule ? this.annotationModule.getColor() : null}
```

Comment:
We should generally try to avoid `null` unless it adds functionality we need. Does `undefined` not work here?

Reply:
`undefined` works here.